Library/Homebrew/test/.bundle/config

```diff
@@ -1,4 +0,0 @@
----
```

Comment:
Is this file still created by bundler itself? My issue is that by removing this we break a manually run `bundle install`

Reply:
Needs to be checked out if we want to run `style` without installing everything else.

Reply:
Ok, thanks.